JavaScript-Runtime-(JSRT)-Reference.md

```diff
@@ -1,12 +1,16 @@
 References of JSRT typedefs, constants, enumerations and APIs. 
 
 **JSRT Typedef References:**
+* [[FetchImportedModuleCallback]]
+* [[FetchImportedModuleFromScriptCallback]]
+* [[NotifyModuleReadyCallback]]
```

Comment:
As an aside, this made me notice that everything else is a `Js` something, including some other callbacks, so perhaps these should be renamed to match the same scheme.

Reply:
We should probably rename them.

Reply:
I guess it's a good thing these APIs are still experimental.